packages/react-server-cli/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "author": "Sasha Aickin",
   "bin": {
-    "react-server-cli": "./bin/react-server-cli"
+    "react-server": "./bin/react-server-cli"
```

Comment:
Unfortunately `rs` generally already exists:

```
$ man rs

RS(1)                     BSD General Commands Manual                    RS(1)

NAME
     rs -- reshape a data array
...
```

FWIW - I'm targeting `react-server start` rather than `react-server startServer`.  Doesn't require the shift key, at least. 😆